telegram/constants.py

```diff
@@ -29,11 +29,12 @@
     * Most of the constants in this module are grouped into enums.
 """
 # TODO: Remove this when https://github.com/PyCQA/pylint/issues/6887 is resolved.
-# pylint: disable=invalid-enum-extension
+# pylint: disable=invalid-enum-extension,invalid-slots
```

Comment:
what's invalid here?

Reply:
`E0238: Invalid __slots__ object (invalid-slots)` for all classes whose base class is `IntEnum`